src/sql/base/parts/editableDropdown/browser/dropdown.ts

```diff
@@ -228,7 +228,6 @@ export class Dropdown extends Disposable implements IListVirtualDelegate<string>
 			}
 			if (this.fireOnTextChange) {
 				this.value = e;
-				this._onValueChange.fire(e);
```

Comment:
Can you verify that this still fires the event correctly when typing in the box? Looking at the code I think we might be skipping the event now since :

1. User types in box
2. VS Input fires onDidChange event
3. This gets called and sets the value
4. In the value setter we check to see if the value is different - but since it's looking directly at _input.value they would be the same. So we'll never fire the onValueChange event.